src/components/Pages/Opportunities/Opportunities.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import Project from '../../Project/Project'
+import TagCluster from '../../TagCluster/TagCluster'
+import { sortByTags, filterBySearch } from '../../../services/utils'
+import Search from '../../Search/Search'
+import Overlay from '../../Project/ProjectOverlay/Overlay'
+import Header from '../../Header/Header'
+
+export default function Opportunities({ onBack }) {
+  const [tags, setTags] = useState(null)
+  const [projects, setProjects] = useState(null)
+  const [searchTerm, setSearchTerm] = useState()
+  const [scrollPosition, setScrollPosition] = useState(0)
+
+  useEffect(() => {
+    fetch('https://unfinished-api.herokuapp.com/api/projects')
+      .then((res) => res.json())
+      .catch((error) => console.log(error))
+      .then((projects) => setProjects(projects))
+    fetch('https://unfinished-api.herokuapp.com/api/tags')
+      .then((res) => res.json())
+      .catch((error) => console.log(error))
+      .then((tags) => setTags(tags))
+  }, [])
+  useEffect(
+    () =>
+      setProjects(
+        (projects) =>
+          tags &&
+          projects &&
+          sortByTags(
+            tags.filter((tag) => tag.applies).map((tag) => tag.text),
+            projects
+          )
+      ),
+    [tags]
+  )
+  useEffect(
+    () => setProjects((projects) => searchTerm && projects && filterBySearch(searchTerm, projects)),
+    [searchTerm]
+  )
+  return (
+    <>
+      <Header title='Opportunities' onBack={onBack} />
+
+      <Search onSearch={onSearch} />
+      <StyledExplanation>
+        {projects &&
+          projects.filter(
```

Comment:
Try `projects?.filter()` instead of `projects && projects.filter()`.
Also keep your JSX clean:
```
{numFoundProjects} out of {projects.length} projects fit your search!
```